raiden/tests/integration/api/test_pythonapi.py

```diff
@@ -262,57 +262,6 @@ def test_transfer_to_unknownchannel(raiden_network, token_addresses):
         )
 
 
-@pytest.mark.skip
-@pytest.mark.parametrize("number_of_nodes", [2])
-@pytest.mark.parametrize("number_of_tokens", [2])
-@pytest.mark.parametrize("settle_timeout", [TEST_TOKEN_SWAP_SETTLE_TIMEOUT])
-def test_token_swap(raiden_network, deposit, token_addresses):
```

Comment:
Do we have any other token swap tests or an issue to create tests?

Reply:
No we don't. This was for the old token swap functionality that has already been removed.
From here and on the plan is to rely on token swaps with the way external xud way. There is an issue for that here: https://github.com/raiden-network/raiden/issues/4628